documentation/book/assembly-customizing-deployments.adoc

```diff
@@ -22,8 +22,10 @@ include::con-customizing-labels-and-annotations.adoc[leveloffset=+1]
 
 include::con-customizing-pods.adoc[leveloffset=+1]
 
+include::con-customizing-containers.adoc[leveloffset=+1]
```

Comment:
Is it worth adding an extra bullet point about Kafka broker environment variables to the "Customizing deployments" assembly? See https://strimzi.io/docs/master/#assembly-customizing-deployments-str

Reply:
I just assumed it would be covered under the `Kafka.spec.kafka`?

Reply:
Ah, yes, you're right.